packages/instantsearch.js/src/widgets/frequently-bought-together/frequently-bought-together.tsx

```diff
@@ -111,6 +111,33 @@ const renderer =
         : undefined
     ) as FrequentlyBoughtTogetherUiProps<Hit>['emptyComponent'];
 
+    const layoutComponent = (
+      templates.layout
+        ? (data) => (
+            <TemplateComponent
+              {...renderState.templateProps}
+              templateKey="layout"
+              rootTagName="fragment"
+              data={{
+                items: data.items,
+                templates: {
+                  item: templates.item
```

Comment:
surely there's always at least a default item template?

Reply:
The default item template is defined in the UI component. At the widget level, the `itemComponent` can be undefined, which will be forwarded as-is to the UI component and be defaulted there.